src/controllers/server.js

```diff
@@ -632,11 +632,12 @@ class Server extends EventEmitter {
                 memory: {
                     total: self.docker.procData.memory_stats.usage,
                     cmax: self.docker.procData.memory_stats.max_usage,
-                    amax: self.json.build.memory * 1000000,
+                    amax: self.json.build.memory * 1000000, //This shoud be 1024 * 1024
```

Comment:
That comment isn't needed is it??

Reply:
The comment was added if you want to change it to binary. 😅  Because it will show ~967mb if limit is 1024mb. Values need to be 100% accurate.